ansible/playbooks/AdoptOpenJDK_Windows_Playbook/README.md

```diff
@@ -0,0 +1,19 @@
+# Setting up Windows Machines
+
+There's a process to setting up Windows machines and getting them connected to Jenkins. If not followed, issues can occur with Jenkins workspaces (See: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1674).
+
+1) Log on to the Windows machine via RDP and run the `ConfigureRemotingForAnsible` commands listed in [main.yml](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Windows_Playbook/main.yml).
+
+2) Run the playbook on the machine, without skipping the 'adoptopenjdk' and 'jenkins' tags. (See [this](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/README.md) for more information).
+
+3) Login to the Jenkins user on the machine via RDP.
+
+4) On the machine, login to ci.adoptopenjdk.net, create a new Node and download/run the `slave-agent.jnlp`.
```

Comment:
I think this step is normally done as jenkins - not sure what might happen (in terms of permissions etc.) if you do it as Administrator. I suppose if you did it as Administrator you might not have to type the administrator credentials as per point 5 :-)
```suggestion
4) On the machine, login to ci.adoptopenjdk.net as the jenkins user, create a new Node and download/run the `slave-agent.jnlp`.
```

Reply:
By ci.adoptopenjdk.net, I mean the Jenkins CI website - where individuals would sign in with the GitHub logins. Point 4 specifies to login to the machine as the Jenkins user :-)
Indeed, logging in as Admin causes the weird folder permissions, which is the whole reason I'm putting this README in :)

Reply:
I've tried to make this clearer as well by putting `ci.adoptopenjdk.net` as a link.

Reply:
I've put that suggestion in the wrong place in the sentence - it should be made clear that you should be logged into the (windows) machine as the jenkins user as opposed to Administrator

Reply:
Covered in step 3. I'm ok with this.